acados/utils/print.c

```diff
@@ -533,6 +566,108 @@ void print_ocp_qp_res(ocp_qp_res *qp_res)
     return;
 }
 
+
+static void int_print_mat_to_file(FILE *file, int row, int col, int *A, int lda)
+{
+    int i, j;
+    for(i=0; i<row; i++)
+    {
+        for(j=0; j<col; j++)
+        {
+            fprintf(file, "%d ", *(A+i+j*lda));
+        }
+        fprintf(file, "\n");
+    }
+    fprintf(file, "\n");
+}
+
+
+
+void print_ocp_qp_in_to_file(FILE *file, ocp_qp_in *qp_in)
```

Comment:
Same as for the QP sol.

Reply:
I decided to just leave a note, stating that these routines should be moved to HPIPM, for now.